web/annotation_layer_builder.js

```diff
@@ -165,7 +170,8 @@ class DefaultAnnotationLayerFactory {
     enableScripting = false,
     hasJSActionsPromise = null,
     mouseState = null,
-    fieldObjectsPromise = null
+    fieldObjectsPromise = null,
+    annotationCanvasMap = null
```

Comment:
Please remember to update https://github.com/mozilla/pdf.js/blob/master/web/interfaces.js as well.